### PR TITLE
Update syntax for Cairo 0.10

### PR DESCRIPTION
**Pull request by frangio**
Update syntax for [Cairo 0.10](https://github.com/starkware-libs/cairo-lang/releases/tag/v0.10.0)

Fixes https://github.com/OpenZeppelin/contracts-wizard/issues/173

### CODE REVIEW

**Comment by ericglau** [comment]
Requires update in `highlightjs-cairo` for comment syntax: https://github.com/OpenZeppelin/highlightjs-cairo/pull/1

**Comment by ericglau** [comment]
@frangio This PR is ready for review.

**Comment by frangio** [comment]
I made an improvement to how functions are formatted that is more aligned with the OpenZeppelin Contracts for Cairo style for 0.10.

**Comment by frangio** [comment]
I can't approve my own pull request, but feel free to go ahead an merge.

I'm assuming we tested that the result compiles.

**Review by ericglau — APPROVED** [review]
Will merge pending release of https://github.com/OpenZeppelin/cairo-contracts/pull/449.

**Comment by ericglau** [comment]
Updated how return values are handled: in most cases functions will return the value from the library call directly, except in cases where the contract should return with a differently named argument.

File: packages/core-cairo/src/contract.ts [hunk 1]
```diff
@@ -31,15 +31,22 @@ export interface Initializer {
...
+ */
+export type PassthroughOption = true | false | 'strict'; 
+
 export interface BaseFunction {
   module?: Module;
   name: string;
   implicitArgs?: Argument[];
   args: Argument[];
   returns?: Argument[];
-  returnValue?: string;
```
Comment (frangio) [inline packages/core-cairo/src/contract.ts:40]: We should probably bump the version of `core-cairo` to communicate this breaking change.

Comment (ericglau) [reply packages/core-cairo/src/contract.ts:40]: I'd consider it a breaking change regardless, due to the Cairo 0.10 syntax.  Will bump from 0.4.0 to 0.5.0 when releasing.

**Comment by ericglau** [comment]
Fixes https://github.com/OpenZeppelin/contracts-wizard/issues/173

**Pull request merged**